src/parkapi_sources/converters/huefner/converter.py

```diff
@@ -0,0 +1,52 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, timezone
+from typing import Any
+
+from openpyxl.cell import Cell
+
+from parkapi_sources.converters.base_converter.push import NormalizedXlsxConverter
+from parkapi_sources.models import SourceInfo
+
+
+class HuefnerPushConverter(NormalizedXlsxConverter):
+    source_info = SourceInfo(
+        uid='huefner',
+        name='PARK SERVICE HÜFNER GmbH & Co. KG',
+        public_url='https://www.ps-huefner.de/parken.php',
+        has_realtime_data=False,
+    )
+
+    purpose_mapping: dict[str, str] = {
+        'Auto': 'CAR',
+        'Fahrrad': 'BIKE',
+    }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # For new required ParkAPI field "purpose"
+        goldbeck_header_rows: dict[str, str] = {
+            'Zweck der Anlage': 'purpose',
+        }
+        self.header_row = {
+            **{key: value for key, value in super().header_row.items() if value not in goldbeck_header_rows.values()},
+            **goldbeck_header_rows,
+        }
+
+    def map_row_to_parking_site_dict(self, mapping: dict[str, int], row: list[Cell]) -> dict[str, Any]:
+        parking_site_dict = super().map_row_to_parking_site_dict(mapping, row)
+
+        for field in mapping.keys():
+            parking_site_dict[field] = row[mapping[field]].value
+
+        if '-00:00' in parking_site_dict['opening_hours']:
```

Comment:
Even simpler: `endswith` :)

Reply:
Yeah, i also thought about ```endswith``` but in the case of opening times like```Mo-Fr 06:00-00:00; Sa 08:00-23:00```, then the IF condition will be ignored and ```00:00``` will not be replaced with ```24:00```, since the string does not end with ```00:00```. Or what do you think?

Reply:
Ah! Ok, you are right, did not think about multiple day. Then it's fine your way.